---
layout: post
title:  "Group I"
date:   2022-04-18T14:25:52-05:00
author: Chichau Miao
categories: ribozyme
---
**[Rfam ID:RF01807](https://rfam.xfam.org/family/RF01807)** <br>

#### click into different sections:

- [Timeline](#timeline)
- [Description](#description)
- [Structure](#structure)
- [Reference](#reference)




## Timeline

***
<html lang="zh-cn">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>

</head>
<style>
   table {
        border: 2px solid #f8f8ff;
        border: 2px solid #767676;
		    border: 2px solid #767676;
		    border-radius: 5px;
		    background-color: #fff;
        }
.timeline {
  list-style: none;
  padding: 10px 0 10px;
  position: relative;
}
.timeline:before {
  top: 0;
  bottom: 0;
  position: absolute;
  content: " ";
  width: 3px;
  background-color: #eeeeee;
  left: 50%;
  margin-left: -1.5px;
}
.timeline > li {
  margin-bottom: 10px;
  position: relative;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li > .timeline-panel {
  width: 46%;
  float: left;
  border: 1px solid #d4d4d4;
  border-radius: 2px;
  padding: 5px;
  position: relative;
  -webkit-box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
}
.timeline > li > .timeline-panel:before {
  position: absolute;
  top: 16px;
  right: -12px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #ccc;
  border-right: 0 solid #ccc;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-panel:after {
  position: absolute;
  top: 16px;
  right: -10px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #fff;
  border-right: 0 solid #fff;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-badge {
  color: #fff;
  width: 25px;
  height: 25px;
  line-height: 40px;
  font-size: 1.4em;
  text-align: center;
  position: absolute;
  top: 16px;
  left: 48.5%;
  margin-left: 0px;
  background-color: #999999;
  z-index: 100;
  border-top-right-radius: 50%;
  border-top-left-radius: 50%;
  border-bottom-right-radius: 50%;
  border-bottom-left-radius: 50%;
}
.timeline > li.timeline-inverted > .timeline-panel {
  float: right;
}
.timeline > li.timeline-inverted > .timeline-panel:before {
  border-left-width: 0;
  border-right-width: 15px;
  left: -15px;
  right: auto;
}
.timeline > li.timeline-inverted > .timeline-panel:after {
  border-left-width: 0;
  border-right-width: 14px;
  left: -14px;
  right: auto;
}
.timeline-badge.primary {
  background-color: #2e6da4 !important;
}
.timeline-badge.success {
  background-color: #3f903f !important;
}
.timeline-badge.warning {
  background-color: #f0ad4e !important;
}
.timeline-badge.danger {
  background-color: #d9534f !important;
}
.timeline-badge.info {
  background-color: #5bc0de !important;
}
.timeline-title {
  margin-top: 0;
  color: inherit;
}
.timeline-body > p,
.timeline-body > ul {
  margin-bottom: 0;
  padding-bottom: 0;
}
.timeline-body > p + p {
  margin-top: 0px;
}

</style>
<!--<div class="container" width: 1026px></div>-->
    <ul class="timeline">
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/6297754" target="_blank" style="font-size:25px;" >1982 </a> Discovery[1]</h4> 
            </div>
            <div class="timeline-body">
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/6757759" target="_blank" style="font-size:25px;" >1982 </a> Determination of shared secondary structure[2]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/3941911" target="_blank" style="font-size:25px;">1986 </a>  The intervening sequence RNA of Tetrahymena is an enzyme[3]</h4>
            </div>
            <div class="timeline-body">
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/3209068" target="_blank" style="font-size:25px;" >1988 </a>  ωG is closely related to the choice of 3' splice site[4]</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/2258934"  target="_blank" style="font-size:25px;">1990 </a>  3D models of  group I intron based on comparative sequence analysis</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/8781224" target="_blank" style="font-size:25px;" >1996 </a> Crystal structure of Tetrahymena P4-P6 domain</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/9841391"  target="_blank" style="font-size:25px;">1998 </a>  Crystal structure of an engineered, active Tetrahymena ribozyme at 5.0 Å resolution</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/15175762" target="_blank" style="font-size:25px;" >2004 </a> Crystal structure of Azoarcus group I intron with both exons</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/15525509"  target="_blank" style="font-size:25px;">2004 </a> Crystal structure of an active Tetrahymena ribozyme</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/15580277" target="_blank" style="font-size:25px;" >2005 </a> Crystal structure of  phage Twort group I ribozyme-product complex</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/16141079"  target="_blank" style="font-size:25px;">2005 </a>Crystal structure of a catalytically active Azoarcus group I intron splicing intermediate</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/34381213" target="_blank" style="font-size:25px;" >2021 </a>Cryo-EM structures of full-length Tetrahymena ribozyme</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/35501384"  target="_blank" style="font-size:25px;">2021 </a>Tetrahymena group I intron at 2.98-Å resolution overall (2.85 Å for the core)</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/17942415" target="_blank" style="font-size:25px;" >2008 </a> Sequence and structure database</h4>
            </div>
          </div>
        </li>
    </ul>

</html>


## Description

***

<font size=4>Group I introns are the first intron ribozymes discovered in the 1980s. They can use exogenous guanosine as a cofactor to catalyze self-cleavage and exon ligation through a two-step translipidation reaction. They are found in genes encoding rRNA, mRNA and tRNA in the nucleus, mitochondria and chloroplast in a variety of organisms from prokaryotes to eukaryotes.</font><br>


## Structure

***

> 2D representation

<font size=4>Secondary structure of the apo L-21 ScaI Tetrahymena ribozyme based on the cryo-EM model, a linear form of the self-splicing intron without its first 21 nucleotides (nts 22–409) 
 </font><br>

<table><tr>
<td>
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
    <title>Document</title>
</head>
<style>
   body {
    width: 100%;
    height: 100vh;
}
   button {
   margin-right: 0px;
}
   .main-container {
    display: flex;
    align-items: left;
    justify-content: center;
    height: 100%;
}
   .zoom-wrapper1 {
    width: 450px;
    height: 400px;
    border: 1px solid #fff;
    display: flex;
    align-items: center;
    justify-content: center;
}
</style>
    <div class="main-container">
        <div class="zoom-wrapper1">
            <div class="zoom-area1">
                <img src="http://127.0.0.1:4000/images/groupIPic/groupI2D.svg" alt="drawing" style="height:160px" />
            </div>
        </div>
    </div>
    <script src="https://timmywil.com/panzoom/demo/panzoom.js"></script>
    <script type='text/javascript'>
      var zoomWraper1 = document.querySelector(".zoom-wrapper1");
      var panzoom1 = Panzoom(document.querySelector(".zoom-area1"), {
      maxScale: 6
      });
      zoomWraper1.addEventListener("wheel", panzoom1.zoomWithWheel);
      panzoom1.zoom(300 / document.querySelector(".zoom-area1 img").height);
      panzoom2.pan(0, 0);
      </script>
</td>
<td>
<link rel="stylesheet" type="text/css" href="https://www.ribocentre.org/css/fornac.css" media="screen" />

<div id='multi_ss'> </div>
       
<meta charset="utf-8">
    <script type='text/javascript' src='https://www.ribocentre.org/js/jquery.js'></script>
    <script type='text/javascript' src='https://www.ribocentre.org/js/d3.js'></script>
    <!--<script type='text/javascript' src='https://www.ribocentre.org/js/fornac.js'></script>-->
    <script type='text/javascript' src='https://taolab.atlasbioinfo.com/rsvdb%2Fjs%2Ffornac.js'></script>
    <script type="text/javascript">
     "use strict"
     function multiContainer() {
        var container = new fornac.FornaContainer("#multi_ss",
                {'applyForce': 1,'editable':'true',  'initialSize':[450,400]});

        container.addRNA('.....................', {'sequence': 'AGCGCCUGGACUUAAAGCCAU'});
        container.addRNA('.............(((...[[[[[[...)))..............]]]]]](((((.[[[[[[))))).....((((((......((((((....)))))).......))))))..]]]]]]', {'avoidOthers': true, 'sequence': 'GGCUUUAAGUUGACGAGGGCAGGGUUUAUCGAGACAUCGGCGGGUGCCCUGCGGUCUUCCUGCGACCGUUAGAGGACUGGUAAAACCACAGGCGACUGUGGCAUAGAGCAGUCCGGGCAGGA'});
    }
    let nc =  multiContainer();
   </script>

</td>
</tr></table><br>

> 3D visualisation

<font size=4>Cryo-EM structures of full-length Tetrahymena ribozyme. The overall structure of the Apo L-21 ScaI Tetrahymena ribozyme was generated from PDB ID:7EZ0 at 3.1 Å resolution. </font>

<table><tr>
<td><img src="http://127.0.0.1:4000/images/groupIPic/groupI3D.png" alt="drawing" style="weight:600px" border=0></td>
<td>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>PDBe Molstar</title>
    <!-- Molstar CSS & JS -->
    <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
    <script src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-1.2.1.js"></script>
        <script>
        function customize()
        {
            viewerInstance.canvas.setBgColor({r:255, g:255, b:255})
        }
        </script>

  <style>
      * {
          margin: 0;
          padding: 0;
          box-sizing: border-box;
      }
      .msp-plugin ::-webkit-scrollbar-thumb {
          background-color: #474748  !important;
      }
      .viewerSection {
        padding-top: 0px;
      }
      .controlsSection {
        width: 300px;
        float: margin-right;
        padding: 10px 0 0 0px;
        margin-right: 10px;
      }
      #myViewer{
        width:450px;
        height: 450px;
        position:relative;  
      }
      
  </style>
  </head>
  <body onload="customize()">
  <!-- Molstar container -->
  <div id="myViewer">
  </div>
  <script>
    var viewerInstance = new PDBeMolstarPlugin();
    var options = {
      moleculeId: '7ez0',
      expanded: false,
      expanded: false,
      hideControls: true
      }
    var viewerContainer = document.getElementById('myViewer');
    viewerInstance.render(viewerContainer, options);
  </script>
  </body>
</html></td>
</tr></table><br>


> Catalytic centre

The active centre of the ribozyme is composed of four layers of base triples.(PDB ID: 7EZ0)<br>

The transition state of first step catalysis.MB binds to the 3’-hydroxyl group of the exogenous guanosine in the first step, functioning as a general base to activate the nucleophile. Another Mg2+ (MA) stabilizes the 3’-leaving group, acting as a general acid. These two metal ions exchange their roles in the second step reaction, of which MA functions as the general base and MB as the general acid.<br>
<table><tr>
<td><img src="http://127.0.0.1:4000/images/groupIPic/groupI-m1.png" alt="drawing" style="width:450px;height:350px" border=1 px></td>
<td><img src="http://127.0.0.1:4000/images/groupIPic/groupI-m2.png" alt="drawing" style="width:450px;height:360px" border=1 px></td>
</tr></table><br>

## Reference

***

**Control of gene expression by a natural metabolite-responsive ribozyme**<br>
Winkler, W., A. Nahvi, A. Roth, J. Collins and R. Breaker <br>
[Nature 428(6980): 281-286.(2004)]((https://www.ncbi.nlm.nih.gov/pubmed/15029187))<br><br>

**Core requirements for glmS ribozyme self-cleavage reveal a putative pseudoknot structure.**<br>
Soukup, G. <br>
[Nucleic acids research 34(3): 968-975.(2006). ]((https://www.ncbi.nlm.nih.gov/pubmed/16464827))<br><br>

**Structural basis of glmS ribozyme activation by glucosamine-6-phosphate.**<br>
Klein, D. and A. Ferré-D'Amaré <br>
[Science (New York, N.Y.) 313(5794): 1752-1756.(2006)]((https://www.ncbi.nlm.nih.gov/pubmed/16990543))<br><br>

**Structural investigation of the GlmS ribozyme bound to Its catalytic cofactor.**<br>
Cochrane, J., S. Lipchock and S. Strobel <br>
[Chemistry & biology 14(1): 97-105.(2007)]((https://www.ncbi.nlm.nih.gov/pubmed/17196404))<br><br>

**Essential role of an active-site guanine in glmS ribozyme catalysis.**<br>
Klein, D., M. Been and A. Ferré-D'Amaré <br>
[Journal of the American Chemical Society 129(48): 14858-14859.(2007)]((https://www.ncbi.nlm.nih.gov/pubmed/17990888))<br><br>

**Mechanism of mRNA destabilization by the glmS ribozyme.**<br>
Collins, J., I. Irnov, S. Baker and W. Winkler <br>
[Genes & development 21(24): 3356-3368.(2007)]((https://www.ncbi.nlm.nih.gov/pubmed/18079181))<br><br>

**Structural and chemical basis for glucosamine 6-phosphate binding and activation of the glmS ribozyme.**<br>
Cochrane, J., S. Lipchock, K. Smith and S. Strobel <br>
[Biochemistry 48(15): 3239-3246.(2009)]((https://www.ncbi.nlm.nih.gov/pubmed/19228039))<br><br>

**The glmS ribozyme: use of a small molecule coenzyme by a gene-regulatory RNA.**<br>
Ferré-D'Amaré, A.<br>
[Quarterly reviews of biophysics 43(4): 423-447.(2010)]((https://www.ncbi.nlm.nih.gov/pubmed/20822574))<br><br>

**Analysis of metal ion dependence in glmS ribozyme self-cleavage and coenzyme binding.**<br>
Klawuhn, K., J. Jansen, J. Souchek, G. Soukup and J. Soukup <br>
[Chembiochem : a European journal of chemical biology 11(18): 2567-2571.(2010)]((https://www.ncbi.nlm.nih.gov/pubmed/21108273))<br><br>

**The glmS riboswitch integrates signals from activating and inhibitory metabolites in vivo.**<br>
Watson, P. and M. Fedor<br>
[Nature structural & molecular biology 18(3): 359-363.(2011)]((https://www.ncbi.nlm.nih.gov/pubmed/21317896))<br><br>

**An expanded collection and refined consensus model of glmS ribozymes.**<br>
McCown, P., A. Roth and R. Breaker <br>
[RNA (New York, N.Y.) 17(4): 728-736.(2011)]((https://www.ncbi.nlm.nih.gov/pubmed/21367971))<br><br>

**The glmS ribozyme cofactor is a general acid-base catalyst.**<br>
Viladoms, J. and M. Fedor <br>
[Journal of the American Chemical Society 134(46): 19043-19049.(2012)]((https://www.ncbi.nlm.nih.gov/pubmed/23113700))<br><br>

**An in vitro evolved glmS ribozyme has the wild-type fold but loses coenzyme dependence.**
Lau, M. W. L. and A. R. Ferré-D Amaré <br>
[Nature Chemical Biology 9(12): 805.(2013)]((https://www.ncbi.nlm.nih.gov/pubmed/24096303))<br><br>

**Activation of the glmS Ribozyme Confers Bacterial Growth Inhibition.**<br>
Schüller, A., D. Matzner, C. Lünse, V. Wittmann, C. Schumacher, S. Unsleber, H. Brötz-Oesterhelt, C. Mayer, G. Bierbaum and G. Mayer <br>
[Chembiochem : a European journal of chemical biology 18(5): 435-440.(2017)]((https://www.ncbi.nlm.nih.gov/pubmed/
28012261))<br><br>

**The GlcN6P cofactor plays multiple catalytic roles in the glmS ribozyme.**<br>
Bingaman, J., S. Zhang, D. Stevens, N. Yennawar, S. Hammes-Schiffer and P. Bevilacqua <br>
[Nature chemical biology 13(4): 439-445.(2017)]((https://www.ncbi.nlm.nih.gov/pubmed/28192411))<br><br>

**A Riboswitch-based Inducible Gene Expression System for Trypanosoma brucei.**<br>
Cruz-Bustos, T., S. Ramakrishnan, C. Cordeiro, M. Ahmed and R. Docampo <br>
[The Journal of eukaryotic microbiology 65(3): 412-421.(2018)]((https://www.ncbi.nlm.nih.gov/pubmed/29265590))<br><br>

**Comprehensive sequence-to-function mapping of cofactor-dependent RNA catalysis in the glmS ribozyme.**<br>
Andreasson, J., A. Savinov, S. Block and W. Greenleaf<br>
[Nature communications 11(1): 1663.(2020)]((https://www.ncbi.nlm.nih.gov/pubmed/32245964))<br><br>

**Targeting glmS Ribozyme with Chimeric Antisense Oligonucleotides for Antibacterial Drug Development.**<br>
Traykovska, M., K. Popova and R. Penchovsky<br>
[ACS synthetic biology 10(11): 3167-3176.(2021)]((https://www.ncbi.nlm.nih.gov/pubmed/34734706))<br><br>








